React/Modules/RCTMultiplicationAnimatedNode.m

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTMultiplicationAnimatedNode.h"
+
+@implementation RCTMultiplicationAnimatedNode
+
+- (void)performUpdate {
+  [super performUpdate];
+  
+  NSArray *inputNodes = self.config[@"input"];
+  if (inputNodes.count > 1) {
+    RCTValueAnimatedNode *parent1 = (RCTValueAnimatedNode *)self.parentNodes[inputNodes[0]];
+    RCTValueAnimatedNode *parent2 = (RCTValueAnimatedNode *)self.parentNodes[inputNodes[1]];
+    if (!parent1 || !parent2) {
+      return;
+    }
+    self.value = @(parent1.value.floatValue * parent2.value.floatValue);
```

Comment:
Seems inconsistent to use floatValue here when you use doubleValue everywhere else.

Also, from what I've seen, there's no reason for all this boxing and unboxing - self.value should just be a double or CGFloat instead of an NSNumber.